### DISCUSSION TITLE
Quick way to find classes and methods by their short name?

### DISCUSSION THREAD

**Discussion by Liyw979** [soot-oss/SootUp#703] [Q&A] [answered]
let say I way to find all `<cinit>` method inside the project.
Can I do some thing like `view.getMethods("<cinit>")?`

**Comment by ericbodden** [comment, marked as answer]
@kadirayk please correct me if I am wrong but I don't think that's possible right now. We would welcome you contributing some functionality, though, that would allow such searches.

**Reply by kadirayk** [reply]
Exactly, currently one needs to specify the full method signature which includes the class type. Right now you can get all the classes `view.getClasses()` and iterate over the classes and get `<clinit>`
